terraform/environments/delius-core/modules/components/delius-microservice/ecs.tf

```diff
@@ -12,9 +12,9 @@ module "container_definition" {
   log_configuration = {
     logDriver = "awslogs"
     options = {
-      "awslogs-group"         = aws_cloudwatch_log_group.delius_core_frontend_log_group.name
```

Comment:
are we ok to remove the log group that was originally created outside this module?

Reply:
Open to thoughts, but I've set this up so that each microservice (ie each call of this module) gets it's own log group. 

New log group created in `terraform/environments/delius-core/modules/components/delius-microservice/cloudwatch.tf` and referenced in the line below this comment

Reply:
marking as resolved. discussed on slack